sr_hardware_interface/include/sr_hardware_interface/tactile_sensors.hpp

```diff
@@ -395,11 +396,40 @@ class UBI0PalmData
   boost::array<uint16_t, 16ul> palm;
 };
 
+class MSTData
+        :
+                public GenericTactileData
+{
+public:
+  MSTData()
+          : GenericTactileData()
+  {
+  };
+
+  explicit MSTData(const GenericTactileData &gtd)
+          : GenericTactileData(gtd.tactile_data_valid, gtd.sample_frequency,
+                               gtd.manufacturer, gtd.serial_number,
+                               gtd.software_version_current,
+                               gtd.software_version_server,
+                               gtd.software_version_modified,
+                               gtd.pcb_version)
+  {
+  };
+
+  ~MSTData()
+  {
+  };
+
+  sr_robot_msgs::MSTPalm sensor_data;
```

Comment:
OK, but `MSTPalm` is a weird name for a type of msg that contains 5 fingers but no palm data.

Reply:
I see your point, what do you suggest the name be?

Reply:
MSTHand perhaps?

Reply:
- For consistency with previous sensors, I would rename [MSTFinger](https://github.com/shadow-robot/sr_common/blob/noetic-devel/sr_robot_msgs/msg/MSTFinger.msg) to `MST` (bear in mind that is a fingertip sensor, not a finger sensor anyway).
- I would also make the `MST` msg have a vector `[]` of taxels, instead of a fixed length array.
- Then I would rename `MSTPalm` to `MSTAll`, for consistency with Biotac etc.

What do you think? @bdenoun I'd like to know your opinion as well. I see yet another msg defined, called [MSTReading](https://github.com/shadow-robot/sr_common/blob/noetic-devel/sr_robot_msgs/msg/MSTReading.msg) what is it used for? Should we use that instead of the MSTFinger msg?

Reply:
We just had the conversation with @rnzenha-s. The MSTReading message was added following the preparations for the beta testing program. It is used to define the topic that is published out of the SPI bridge. But you are right when highlighting that the message that I created definitely doesn't follow any naming convention we seemt to have for the other sensors... :/ Sorry about that!! So yeah, it would make sense to use (and rename) the one I made a PR for. If we decide to do that, we will need to update the sr_mst repository as well then. Shall I prepare the PR?